CONTRIBUTING.md

```diff
@@ -3,8 +3,26 @@
 Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
 documentation, we greatly value feedback and contributions from our community.
 
-Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
-information to effectively respond to your bug report or contribution.
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary information to effectively respond to your bug report or contribution.
+
+## Getting Started
+
+If you're looking to join this wonderful wave of collaboration, here's how to begin:
+
+1. Take a look at the open issues to see what needs attention.
```

Comment:
sort of a nit on format, can you make each bullet an action? It's ok to use 1-2 sentences i think

1. Look at issues
2. For an unassigned issue:
   - Important - comment before you start, to indicate your interest
   - Wait for a maintainer ... this prevents...
3. If there's an issue you're interested in that's already assigned:
     - Check last activity...
     - If no recent updates, comment...
            * example "Hi!..."
      
      I know this seems pendantic, but I think it'll help with clarity